src/main/java/io/appform/dropwizard/sharding/dao/LookupDao.java

```diff
@@ -473,11 +487,10 @@ public List<T> scatterGather(DetachedCriteria criteria) {
                 .mapToObj(shardId -> {
                     try {
                         val dao = daos.get(shardId);
+                        val opContext= Select.<T, List<T>>builder()
+                            .getter(dao::select).criteria(criteria).build();
```

Comment:
can scatterGather be made a separate operation?

Reply:
Scattergather is the command perform across shard which internally perform select operation on each shard. So Cannot have different operation for scattergather.